src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -180,13 +185,19 @@ export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
     }
 
     if (cartError) {
-      if (cartError instanceof SessionError) {
-        clearCartError();
-        expireSession();
-        showErrorAlert(cartError, () => {
-          navigation.navigate("CampaignLocationsScreen");
-        });
-        return;
+      switch (true) {
```

Comment:
should we have a default case? 🤔  would the code ever go there?

Reply:
thats true... hmm 🤔  currently if we don't put in the default case, whats the current behaviour? 

I think putting in a default case might not work, cause it might skip the if checks below >< I went to reread everything HAHAHA

Reply:
After reading through the code again, I think putting no `default` case makes more sense.

Without a `default` case, if `cartError` does not match any of the `Error` types, execution will proceed to evaluate the `if` block below!

We can put a `default` case with nothing inside, but that is just a waste of space 😂

Reply:
Yeah.... HAHAHA. Okok we have the same understanding of how the code runs HAHAHA. Ok then I guess let's leave it as is HAHAHA

Reply:
I wrote a snippet and it works as intended:

- If `cartError` matches any of the errors in the switch-case, it will go into that branch.
- Else, the function will continue to evaluate the if block below

Reply:
Nice, thanks for checking! ^^